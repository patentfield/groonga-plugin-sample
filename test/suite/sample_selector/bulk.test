register functions/sample

table_create Values TABLE_NO_KEY
column_create Values value COLUMN_SCALAR ShortText

load --table Values
[
{"value": "a"},
{"value": "b"},
{"value": "c"}
]

select Values --filter 'all_records() && sample_selector("value", {"includes": "a,c"})'
