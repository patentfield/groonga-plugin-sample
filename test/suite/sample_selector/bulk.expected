register functions/sample
[[0,0.0,0.0],true]
table_create Values TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Values value COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Values
[
{"value": "a"},
{"value": "b"},
{"value": "c"}
]
[[0,0.0,0.0],3]
select Values --filter 'all_records() && sample_selector("value", {"includes": "a,c"})'
[[0,0.0,0.0],[[[2],[["_id","UInt32"],["value","ShortText"]],[1,"a"],[3,"c"]]]]
